### PR TITLE
Alerts and source info

### PR DESCRIPTION
**Pull request by walterwhite1337**
Added a bootstrap-like info alert about source code

### CODE REVIEW

File: css/cloudsun.css [hunk 1]
```diff
@@ -599,4 +634,7 @@ a {
 .card-body {
   flex: 1 1 auto;
   padding: 1rem;
-}
\ No newline at end of file
+}
+
+/* The end.
```
Comment (trlandrew) [inline css/cloudsun.css:639]: pls remov this :( css then go brokey broke

Comment (walterwhite1337) [reply css/cloudsun.css:639]: qhar

Comment (walterwhite1337) [reply css/cloudsun.css:639]: done

**Comment by LNLenost** [comment]
Ill translate this asap

**Pull request merged**